layer1/layer1.yaml

```diff
@@ -52,12 +52,12 @@ components:
           description: >-
             Enable promiscuous mode if supported.
           type: boolean
-          default: false
+          default: true
         mtu:
           description: >-
             Set the maximum transmission unit size if supported.
           type: integer
-          default: 1500
+          default: 1518
```

Comment:
This doesn't appear to be the common default.

Reply:
Changed